src/commissaire_service/storage/__init__.py

```diff
@@ -135,12 +135,12 @@ def on_save(self, message, model_type_name, model_json_data):
         :type model_type_name: str
         :param model_json_data: JSON representation of a model
         :type model_json_data: str
-        :returns: full JSON representation of the model
-        :rtype: str
+        :returns: full dict representation of the model
+        :rtype: dict
         """
         model_instance = self._build_model(model_type_name, model_json_data)
         saved_model_instance = self._manager.save(model_instance)
-        return saved_model_instance.to_json()
+        return saved_model_instance.__dict__
```

Comment:
Hrm, yeah, it would. It would then be up to the HTTP ``handlers`` to hide them which isn't ideal. I think I'll add in a ``to_dict`` method which will provide a sanitized dictionary. Will update this PR.